gensim/models/nmf_pgd.pyx

```diff
@@ -47,8 +44,9 @@ def solve_h(double[:, ::1] h, double[:, :] Wt_v_minus_r, double[:, ::1] WtW, dou
 
     for sample_idx in prange(n_samples, nogil=True):
```

Comment:
Don't you should add some options in setup.py to enable this "magic" ?

Reply:
not mandatory, current "magic" works even without a compiler flags :) 
I guess it's a good idea to enable compiler options later for all `.pyx`, but not now.

Reply:
Of course, it will work. but will it work parallel ?

Reply:
Well, works parallel on my machine. :)

Reply:
I agree, we'll deal with this later